src/Microsoft.PowerShell.Commands.Utility/commands/utility/sort-object.cs

```diff
@@ -2,14 +2,19 @@
 Copyright (c) Microsoft Corporation.  All rights reserved.
 --********************************************************************/
 
+using System.Collections.Generic;
 using System.Management.Automation;
 
 namespace Microsoft.PowerShell.Commands
 {
     /// <summary>
     /// 
     /// </summary>
-    [Cmdlet("Sort", "Object", HelpUri = "https://go.microsoft.com/fwlink/?LinkID=113403", RemotingCapability = RemotingCapability.None)]
```

Comment:
Also, in the Pester tests file, tab indentation was done using tabs instead of spaces which are used everywhere else for the indentation, I corrected this mistake. VS Code by default applies the indentation of the file, however that assumes you are just entering your code into the editor and not copy/pasting code from elsewhere. It is better for the indentation to be done consistently using spaces, so this was applied to the Pester tests file.